ironfish/src/memPool/memPool.test.ts

```diff
@@ -50,18 +50,31 @@ describe('MemPool', () => {
   describe('get', () => {
     const nodeTest = createNodeTest()
 
-    it('returns transactions from the node mempool', async () => {
+    it('returns transactions from the node mempool sorted by fees', async () => {
       const { node } = nodeTest
       const { accounts, memPool } = node
       const accountA = await useAccountFixture(accounts, 'accountA')
       const accountB = await useAccountFixture(accounts, 'accountB')
-      const { transaction } = await useBlockWithTx(node, accountA, accountB)
-
-      await memPool.acceptTransaction(transaction)
+      const { transaction: firstTransaction } = await useBlockWithTx(node, accountA, accountB)
```

Comment:
this is such a tiny nit, but in other places we usually name this with a suffix like the accounts above so `transactionA` or `transaction1` and `transactionB` or `transaction2`